data/tutor.yaml

```diff
@@ -14,18 +14,23 @@
 #   availability: MWF, 9AM-9PM
 
 - name: Emily Gubski
-  lastmod: 2021-11-21
-  courses: ["CPSC103"]
-  email: emily.gubski@ubc.ca
+  lastmod: 2022-01-22
+  courses: [CPSC103, Highschool CS and Math, MCAT CARS Prep]
```

Comment:
The "Highschool CS and Math" and "MCAT CARS Prep" will break the tutoring filtering functionality, but only for those two elements. (The spaces and the current filtering code don't play nice - this is the same reason why we had to switch from "CPSC XXX" to "CPSCXXX".)

If you want, I can try to fix the filtering script to handle this case. Otherwise, maybe we could move it to the bio section?

Reply:
Yes, we can remove it. Thanks for reminding me about this issue!